### ISSUE TITLE
EBE textures not changing in latest GIT compile

### ISSUE THREAD

**Issue by Ratall** [TheGameCreators/GameGuruRepo#111]
I was having a play with the latest GIT version**

and decided to use EBE to fill an odd sized hole between entities. Unfortunately this happened.
![ebe_edit_mode](https://user-images.githubusercontent.com/35710601/37130371-3e219a04-22d7-11e8-8ceb-157eda89d909.PNG)
Above as appears when editing.
![ebe_placed](https://user-images.githubusercontent.com/35710601/37130414-6e9ca8f4-22d7-11e8-997c-35df05e9f390.PNG)
But this is what your get after pressing E

In summary : You can change the texture when editing in EBE but when you exit the changed texture  reverts back to what it was originally.
**Dated 7th March 2018 7:46

**Comment by plemsoft** [comment]
Tested in 2018.04.14 DX11 , and it worked fine , i was not able to reproduce this.

**Comment by Ratall** [comment]
Sorry for the delay in responding I tried to replicate the problem over the weekend but could not. I guess it was either something wrong with my system at the time of it has been resolved.  thank you for your time.
Rick
